pkgs/by-name/fu/furmark/package.nix

```diff
@@ -0,0 +1,84 @@
+{ autoPatchelfHook
+, copyDesktopItems
+, fetchzip
+, lib
+, libGL
+, libGLU
+, makeDesktopItem
+, makeWrapper
+, stdenv
+, vulkan-loader
+}:
+
+let
+  description = "OpenGL and Vulkan Benchmark and Stress Test";
+  versions = {
+    "x86_64-linux" = "2.2.0.0";
+    "i686-linux" = "2.0.16";
+  };
```

Comment:
non-blocking, but 2.3.0.0 is out. Also, I'm not even sure we should support i686-linux, given that it's [not listed on the download page](https://www.geeks3d.com/furmark/downloads/)